### PR TITLE
Added language switcher

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Allows language to be switched for social surveys
Currently hard coded to en/cy

### How to review 
load `test_language.json` and use the switcher in the top right

### CODE REVIEW

**Comment by jonnyshaw89** [comment]
@boxadesign can you have a look at the styles here after the new header is in?

**Comment by dcdarrell9** [comment]
Will this need to be able to switch at different points? Like introduction/summary, if so would it be worth adding to the test schema

**Comment by jonnyshaw89** [comment]
@dcdarrell9 you mean I should test that the switcher works on each page?

File: app/jinja_filters.py [hunk 1]
```diff
@@ -373,6 +373,17 @@ def max_value(a, b):
         raise Exception(msg)
 
 
+@blueprint.app_template_filter()
+def language_urls(languages, current_language):
+    switcher = []
+    for language in languages:
+        if language[0] == current_language:
+            continue
```
Comment (bitdivision) [inline app/jinja_filters.py]: Not a big deal, but could this be made simpler?
```
return [l for l in languages if l[0] != current_language]
```

**Review by bitdivision — COMMENTED** [review]
Looks good, will test in a moment.

File: app/jinja_filters.py (hunk 1 above)
Comment (jonnyshaw89) [reply app/jinja_filters.py]: ah yeh, I refactored some logic out and was lazy

**Review by bitdivision — APPROVED** [review]
Just need to sort out `ru_name`

**Review by dcdarrell9 — APPROVED** [review]

**Pull request merged**